### PR TITLE
Update description of three-body and surface reactions

### PR DESCRIPTION
**Pull request by ischoegl**
* Add reference to implict third body collider
* Revise section describing surface and sticking reactions

See also Cantera/cantera#1249

### CODE REVIEW

File: pages/science/kinetics.rst [hunk 1]
```diff
@@ -384,14 +390,28 @@ Surface reactions can be defined in the CTI format using the
...
-Sticking Coefficients
-~~~~~~~~~~~~~~~~~~~~~
+In YAML, the surface reaction ``type`` defaults to ``interface-Arrhenius``, where
+the rate expression uses the
+:ref:`Arrhenius <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-interface-Arrhenius>`__
+parameterization. As an alternative, Cantera also supports the
+``interface-Blowers-Masel`` surface reaction ``type``, which uses the
+:ref:`Blowers-Masel <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-interface-Blowers-Masel>`__
+parameterization *(New in Cantera 2.6)*. The prefix ``interface-`` is
+not required as it is inferred automatically.
```
Comment (bryanwweber) [inline pages/science/kinetics.rst:404]: I think it's probably better not to specify that the prefix is not required; explicit is better than implicit, and this reads as a bit confusing to me ("where is the `interface-` not required?").

**Review by bryanwweber — APPROVED** [review]
Thanks @ischoegl just a couple small comments for discussion.

File: pages/science/kinetics.rst [hunk 2]
```diff
@@ -384,14 +390,28 @@ Surface reactions can be defined in the CTI format using the
...
+.. TODO: Update the link once version 2.6 is released
 
-Sticking Coefficients
-~~~~~~~~~~~~~~~~~~~~~
+In YAML, the surface reaction ``type`` defaults to ``interface-Arrhenius``, where
+the rate expression uses the
+:ref:`Arrhenius <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-interface-Arrhenius>`__
+parameterization. As an alternative, Cantera also supports the
+``interface-Blowers-Masel`` surface reaction ``type``, which uses the
+:ref:`Blowers-Masel <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-interface-Blowers-Masel>`__
```
[the remark this answers is not in the document]
Comment (ischoegl) [reply pages/science/kinetics.rst]: Including both.

File: pages/science/kinetics.rst (hunk 1 above)
Comment (ischoegl) [reply pages/science/kinetics.rst]: I agree that it can be omitted here.

**Comment by ischoegl** [comment]
Fixed links ... I believe this is ready.

**Comment by ischoegl** [comment]
@bryanwweber … are you :+1: with the updates?

**Pull request merged**